ethers-solc/src/config.rs

```diff
@@ -348,7 +348,7 @@ impl ProjectPathsConfig {
                     // (`<name>/=.../contracts`) and the stripped import also starts with
                     // `contracts`
                     if let Ok(adjusted_import) = stripped_import.strip_prefix("contracts/") {
-                        if r.path.ends_with("contracts/") && !lib_path.exists() {
+                        if r.path.ends_with("/contracts/") && !lib_path.exists() {
```

Comment:
let's add a note here with the openzeppelin-contracts as example

I need as much context as possible for all these rules

Reply:
Ok! Since it applies to more than oz, I wanted to keep it general.
we want to prevent matching on directories that end in things like `-contracts`, `_contracts`, `.contracts`